e2e/tests/shared/shared.go

```diff
@@ -73,7 +73,7 @@ type Test struct {
 	Cleanup           bool
 }
 
-func SetupTestWithDefaults(testName string) (*Test, error) {
+func SetupTestWithDefaults(testName, utiliyManaged string) (*Test, error) {
```

Comment:
I think passing in `utiliyManaged` as a bool makes a bit more sense than as a string.

```suggestion
func SetupTestWithDefaults(testName string, argoManagedUtilities bool) (*Test, error) {
```